day_7/ceasar_cipher_attempt.rb

```diff
@@ -0,0 +1,41 @@
+class CeasarCipher
+
+  def initialize
+  end
+
+  def encode(word, shift_num)
+    #make aplphabet into array
+    alphabet = 'abcdefghijklmnopqrstuvwxyz'.split('')
+    #make encoded alphabet array
+    encoded_alphabet = alphabet.unshift(alphabet.last(shift_num + 1)).flatten(1)
+
```

Comment:
Interesting strategy here! So what I am gathering is that you are making a second alphabet array with all of the letters shifted, then you grab the letters of the same index... you may be able to lean on just your original alphabet array here- you could do the 'shift' on your index, but you'll need to account for when that shift takes things beyond the number of elements in your array (like when a z gets shifted several letters).